packages/web-components/src/components/footer/legal-nav.ts

```diff
@@ -67,23 +67,35 @@ class C4DLegalNav extends StableSelectorMixin(LitElement) {
   }
 
   render() {
-    return html`
-      <nav class="${prefix}--legal-nav">
-        <div class="${this._getTypeClass()}">
-          <ul>
-            <slot></slot>
-          </ul>
-          <div class="${prefix}--adjunct-links__container">
-            <div class="${prefix}--adjunct-links__row">
-              <div class="${prefix}--adjunct-links__col">
+    return this.size !== FOOTER_SIZE.MICRO
+      ? html`
+          <nav class="${prefix}--legal-nav">
+            <div class="${this._getTypeClass()}">
+              <ul>
+                <slot></slot>
+              </ul>
+              <slot name="locale"></slot>
+            </div>
+            <div class="${prefix}--adjunct-links__container">
```

Comment:
```suggestion
          <nav class="${c4dPrefix}--legal-nav">
            <div class="${this._getTypeClass()}">
              <ul>
                <slot></slot>
              </ul>
              <slot name="locale"></slot>
            </div>
            <div class="${c4dPrefix}--adjunct-links__container">
```

Should be our prefix, right?

Reply:
yeah good call!